charts/agent/templates/configmap-deployment.yaml

```diff
@@ -43,6 +43,18 @@ data:
 {{- if .Values.global.sysdig.tags }}
     tags: {{ include "agent.tags" . }}
 {{- end }}
+{{- if and (or (.Values.proxy.httpProxy | default .Values.global.proxy.httpProxy) (.Values.proxy.httpsProxy | default .Values.global.proxy.httpsProxy)) (not .Values.sysdig.settings.http_proxy) }}
```

Comment:
Just as a note, with this new method it will not be possible to use both the `global.proxy` path and supply values for the Agent's `http_proxy.ssl_verify_certificate` or `http_proxy.ca_certificate` fields, as those need to be put in `sysdig.settings.http_proxy`.

This config:
```yaml
global:
  proxy:
    httpProxy: http://192.168.10.2:3128
    httpsProxy: http://192.168.10.2:3128
agent:
  sysdig:
    settings:
      http_proxy:
        ssl_verify_certificate: true
        ca_certificate: /usr/proxy/proxy.crt
```

results in:
```yaml
http_proxy:
  ca_certificate: /usr/proxy/proxy.crt
  ssl_verify_certificate: true
```